src/game/ObjectMgr.cpp

```diff
@@ -1975,6 +1975,38 @@ PetLevelInfo const* ObjectMgr::GetPetLevelInfo(uint32 creature_id, uint32 level)
     return &itr->second[level - 1];                         // data for level 1 stored in [0] array element, ...
 }
 
+PetFamilyStatMods const* ObjectMgr::GetPetFamilyStatMods(uint32 family) const
+{
+    // Loading levels data
+    //                                                   0      1     2     3
+    QueryResult* result  = WorldDatabase.Query("SELECT family, healthModifier, armorModifier, damageModifier FROM pet_familystats");
+
+    PetFamilyStatMods* pFamilyMods;
```

Comment:
Could it maybe also have something to do with the fact that I in the table still have an even older naming style of hMod, aMod, dMod, where I here in the core state it as healthMod, armorMod, damageMod? .... I don't sleep nearly enough...